docs/ref/contrib/gis/gdal.txt

```diff
@@ -1469,9 +1469,9 @@ blue.
         ``GDT_UInt32``, ``GDT_Int32``, ``GDT_Float32``, ``GDT_Float64``,
         ``GDT_CInt16``, ``GDT_CInt32``, ``GDT_CFloat32``, and ``GDT_CFloat64``.
 
-    .. method:: data(data=None, offset=None, size=None)
+    .. method:: data(data=None, offset=None, size=None, shape=None)
 
-        .. versionadded:: 1.9
```

Comment:
We remove the annotations after two releases so the 1.9 annotation should remain. The changed annotation should appear at the bottom of the method with a description of what changed. Is it worth a mention in the release notes as well?